ui/app/common/obs/views/showObservation.html

```diff
@@ -25,10 +25,10 @@
                        'value-text-only': !(observation.concept.dataType ==='Text' && print)
                        }"
               ng-if="::(observation.isFormElement() && !observation.isImageConcept() && !observation.isVideoConcept())">
-            <pre class="left obs-label" title="{{::observation.getDisplayValue().substring(observation.getDisplayValue().lastIndexOf('__')+2) | titleTranslate}}"
-                ng-if="configIsObservationForImages && observation.getDisplayValue().lastIndexOf('__')>-1">{{::observation.getDisplayValue().substring(observation.getDisplayValue().lastIndexOf('__')+2) | titleTranslate}}</pre>
-            <pre class="left obs-label" title="{{::observation.getDisplayValue() | titleTranslate}}"
-                ng-if="configIsObservationForImages && observation.getDisplayValue().lastIndexOf('__')===-1">{{::observation.getDisplayValue() | titleTranslate}}</pre>
+            <span class="left obs-label-file" title="{{::observation.getDisplayValue().substring(observation.getDisplayValue().lastIndexOf('__')+2) | titleTranslate}}"
+                ng-if="configIsObservationForImages && observation.getDisplayValue().lastIndexOf('__')>-1">{{::observation.getDisplayValue().substring(observation.getDisplayValue().lastIndexOf('__')+2) | titleTranslate}}</span>
```

Comment:
This is a really long if() statement. Can we extract this into observation class, and add a more readable method name.. like observation.isValidImage( ) or something like that, which is more meaningful as to WHY this if statement exists.